dcs/triggers.py

```diff
@@ -8,13 +8,19 @@
 
 
 class TriggerZone:
-    def __init__(self, _id, position: mapping.Point, radius=1500, hidden=False, name=""):
+    def __init__(self, _id, position: mapping.Point, radius=1500, hidden=False, name="", color=None, properties={}):
         self.id = _id
         self.radius = radius
         self.position = copy.copy(position)
         self.hidden = hidden
         self.name = name
-        self.color = {1: 1, 2: 1, 3: 1, 4: 0.15}  # TODO color attributes
```

Comment:
You can simplify this even more:
```
self.color = color if color is not None else {1: 1, 2: 1, 3: 1, 4: 0.15}
```

Also you forgot properties:
```
self.properties = properties if properties is not None else {}
```

Reply:
Done.